src/game-data/effects/cards/1-0-136.ts

```
@@ -0,0 +1,16 @@
+import { Effect } from '@/game-data/effects/engine/effect';
+import { EffectHelper } from '@/game-data/effects/engine/helper';
+import { System } from '@/game-data/effects/engine/system';
+import type { CardEffects, StackWithCard } from '@/game-data/effects/schema/types';
+
+export const effects: CardEffects = {
+  checkDrive: (stack: StackWithCard) =>
+    stack.processing.owner.id === stack.source.id &&
+    stack.processing.owner.opponent.hand.length > 0,
+  onDrive: async (stack: StackWithCard) => {
+    await System.show(stack, '迷子', '手札を1枚破壊');
+    EffectHelper.random(stack.processing.owner.opponent.hand).forEach(card =>
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)